spec/features/event_spec.rb

```diff
@@ -273,20 +273,30 @@
   end
 
   describe "edit page" do
-    it "should preselect the event type" do
+    it "should not be possible to visit as pupil" do
```

Comment:
Test case for logged-out user is missing. 
it "should not be possible to visit as logged-out user"